internal/attestation/snp/validator.go

```diff
@@ -58,10 +58,13 @@ func (v *Validator) OID() asn1.ObjectIdentifier {
 func (v *Validator) Validate(attDocRaw []byte, nonce []byte, peerPublicKey []byte) (err error) {
 	v.logger.Info("Validate called", "nonce", hex.EncodeToString(nonce))
 	defer func() {
+		// Note: We tolerate Validate() to fail for validators without matching configuration. Only 1 of n validators has to succeed.
+		// Thus the error handling of the validators takes place in the calling function verifyEmbeddedReport().
```

Comment:
Please make `verifyEmbeddedReport` log the combined error at ERROR level. Otherwise we end up in the situation again that motivated #1027, where connection attempts are silently failing.

Reply:
Discussed in private: the errors are reproted as expected.

Reply:
I double checked the logging mechanisms and it works fine on the coordinator as well as on the local attestation validation. However it might be reasonable to add better formatting to the joined error message.